tensorflow_similarity/models/contrastive_model.py

```diff
@@ -51,25 +52,90 @@
 )
 
 
+def get_projector(input_dim, dim=512, activation="relu", num_layers: int = 3):
+    inputs = tf.keras.layers.Input((input_dim,), name="projector_input")
+    x = inputs
+
+    for i in range(num_layers - 1):
+        x = tf.keras.layers.Dense(
+            dim,
+            use_bias=False,
+            kernel_initializer=tf.keras.initializers.LecunUniform(),
+            name=f"projector_layer_{i}",
+        )(x)
+        x = tf.keras.layers.BatchNormalization(epsilon=1.001e-5, name=f"batch_normalization_{i}")(x)
```

Comment:
is epsilon=1.001e-5 well-established? should it be a constant somewhere to be used by all batchnorm layers?

Reply:
This is used to be as close as possible to the values used by the paper. I'll add it as a constant to make it consistent everywhere.